app/models/miq_server/environment_management.rb

```diff
@@ -134,7 +134,7 @@ def check_disk_usage(disks)
                          when '/var/log/audit'   then 'evm_server_var_log_audit_disk_high_usage'
                          when '/var/www/miq_tmp' then 'evm_server_miq_tmp_disk_high_usage'
                          when '/tmp'             then 'evm_server_tmp_disk_high_usage'
-                         when %r{pgsql/data}     then 'evm_server_db_disk_high_usage'
+                         when %r{lib/pgsql}      then 'evm_server_db_disk_high_usage'
```

Comment:
There isn't actually a change in the structure, just the mountpoint. The full path to the data directory was (and still is) `/var/opt/rh/rh-postgresql95/lib/pgsql/data` I just wanted to make the regex a bit more specific than `%r{/pgsql}`